Bryn Mawr Classical Review.js

```diff
@@ -59,6 +76,8 @@ function doWeb(doc, url) {
 	Zotero.Utilities.processDocuments(arts, function(doc) {
 		var item = new Zotero.Item("journalArticle");
 		var title = doc.evaluate('//h3/i', doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext().textContent;
```

Comment:
another cleanup request:
replace `nsResolver` with `null` in all instances of `doc.evaluate` and delete line 41-44 above, i.e.

```
var ns = doc.documentElement.namespaceURI;
 	var nsResolver = ns ? function(prefix) {
 		if (prefix == 'x') return ns; else return null;
 	} : null;
```